source/draw/Color.ooc

```diff
@@ -25,6 +25,9 @@ ColorMonochrome: cover {
 	normalized ::= this y as Float / 255
 	init: func@ (=y)
 	init: func@ ~default { this init(0) }
+	init: func@ ~uint16 (i: UInt16) { this init(i as UInt8) }
+	init: func@ ~uint32 (i: UInt32) { this init(i as UInt8) }
+	init: func@ ~uint64 (i: UInt64) { this init(i as UInt8) }
```

Comment:
What happens if you have a `UInt` and call one of these? Just curious. Does rock cast to float anyway?